src/editor.js

```diff
@@ -886,11 +886,8 @@ function setupWorkspace(data, callback) {
         renderContent('blocks');
     }
 
-
-    // View or edit project details menu item
```

Comment:
The concept of project ownership is lost in Solo. The projectData.yours is deprecated since it has no meaning in the current context.